### PR TITLE
Create shareduserprofile.md

### PR DESCRIPTION
**Pull request by adimitui**
added resource page for sharedUserProfile

Schema update PR: 5282485

### CODE REVIEW

File: api-reference/beta/api/outboundshareduserprofile-list.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+}
+-->
+``` http
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+{
+    "value": [
+        {
+            "@odata.type": "#Microsoft.DirectoryServices.inboundSharedUserProfiles",
```
Comment (kakagapo) [inline api-reference/beta/api/outboundshareduserprofile-list.md:78]: This is the response corresponding to the inboundSharedUserProfiles type. Can you please fix it ?

File: api-reference/beta/resources/outboundSharedUserProfile.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+|:---|:---|:---|
+|[List outboundSharedUserProfile](../api/outboundSharedUserProfile-list.md)|[outboundSharedUserProfile](../resources/outboundSharedUserProfile.md) collection|List the AAD users whose profile data is shared with external tenants.|
+|[Get outboundSharedUserProfile](../api/outboundSharedUserProfile-get.md)|[outboundSharedUserProfile](../resources/outboundSharedUserProfile.md)|Read the properties and relationships of a [outboundSharedUserProfile](../resources/outboundSharedUserProfile.md) object.|
+
+## Properties
+|Property|Type|Description|
+|:---|:---|:---|
+| userId | String | Object ID of the external user whose profile data is shared with the current tenant. |
+| userPrincipalName | String | The user principal name (UPN) of the external user. |
+| externalTenantId | String | ID of the external tenant with user profile access. |
```
Comment (kakagapo) [inline api-reference/beta/resources/outboundSharedUserProfile.md:29]: The schema for this type was changed. Now there is a navigation property that points to TenantReference entities.

File: api-reference/beta/resources/outboundSharedUserProfile.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  "keyProperty": "id",
+  "@odata.type": "microsoft.graph.tenantRelationshipRoot",
+  "openType": false
+}
+-->
+``` json
+{
+  "id":"ZAMkAAIAAAoZDOFAAA=",
+  "userId":"c228b2ae-c4fb-4eda-9620-7e73dddd1cac",
+  "externalTenantId":"62bfb458-9474-4c44-896b-b30942d055f0"
```
Comment (kakagapo) [inline api-reference/beta/resources/outboundSharedUserProfile.md:47]: This has to be fixed.

File: api-reference/beta/resources/shareduserprofile.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
+---
+title: "sharedUserProfile resource type"
+description: "Represents an AAD user from an external AAD tenant whose profile data is shared with the current tenant."
+author: "adimitui"
+ms.localizationpriority: medium
+ms.prod: "directory-management"
+doc_type: resourcePageType
+---
+
+# sharedUserProfile resource type
```
Comment (kakagapo) [inline api-reference/beta/resources/shareduserprofile.md:10]: This file has to be removed. There is nothing named sharedUserProfile

File: api-reference/beta/resources/userprofilesharedwithexternaltenant.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
+---
+title: "userProfileSharedWithExternalTenant resource type"
+description: "For an AAD user, represents the profile data that is shared with an external AAD tenant (other than the user's home tenant). This does not include profile sharing info for non-AAD users (for example, MSA users)."
+author: "adimitui"
+ms.localizationpriority: medium
+ms.prod: "directory-management"
+doc_type: resourcePageType
+---
+
+# userProfileSharedWithExternalTenant resource type
```
Comment (kakagapo) [inline api-reference/beta/resources/userprofilesharedwithexternaltenant.md:10]: This file has to be removed too.

File: doc-stubs/api/administrativeunit-delete.md [hunk 6]
```diff
@@ -0,0 +1,71 @@
+---
+title: "Delete administrativeUnit"
+description: "Deletes an administrativeUnit object."
+author: "**TODO: Provide Github Name. See [topic-level metadata reference](https://msgo.azurewebsites.net/add/document/guidelines/metadata.html#topic-level-metadata)**"
+ms.localizationpriority: medium
+ms.prod: "**TODO: Add MS prod. See [topic-level metadata reference](https://msgo.azurewebsites.net/add/document/guidelines/metadata.html#topic-level-metadata)**"
+doc_type: apiPageType
+---
+
+# Delete administrativeUnit
```
Comment (kakagapo) [inline doc-stubs/api/administrativeunit-delete.md:10]: Lots of files here are not related to this change. Maybe we can remove them?

**Review by kakagapo — CHANGES_REQUESTED** [review]

**Review by pawaitengineer — COMMENTED** [review]
`doc-stubs` and `changelog-stubs` do not need to be checked in as is.

File: api-reference/beta/api/outboundshareduserprofile-list.md (hunk 1 above)
Comment (adimitui) [reply api-reference/beta/api/outboundshareduserprofile-list.md:78]: Fixed to "#Microsoft.DirectoryServices.outboundSharedUserProfiles"

File: api-reference/beta/resources/shareduserprofile.md (hunk 4 above)
Comment (adimitui) [reply api-reference/beta/resources/shareduserprofile.md]: Removed file

File: doc-stubs/api/administrativeunit-delete.md (hunk 6 above)
Comment (adimitui) [reply doc-stubs/api/administrativeunit-delete.md]: Removed entire doc-stubs and changelog-stubs folders

File: api-reference/beta/resources/userprofilesharedwithexternaltenant.md (hunk 5 above)
Comment (adimitui) [reply api-reference/beta/resources/userprofilesharedwithexternaltenant.md]: Removed

File: api-reference/beta/resources/outboundSharedUserProfile.md (hunk 2 above)
Comment (adimitui) [reply api-reference/beta/resources/outboundSharedUserProfile.md:31]: Added navigation property that points to tenantReference entity

File: api-reference/beta/resources/outboundSharedUserProfile.md (hunk 3 above)
Comment (adimitui) [reply api-reference/beta/resources/outboundSharedUserProfile.md:51]: fixed sample json output to consist of userId, userPrincipalName, and externalTenantId

**Comment by adimitui** [comment]
@pawaitengineer: Performed requested changes, including removing doc-stubs and changelog-stubs folder. The content is ready for a re-review

**Review by FaithOmbongi — CHANGES_REQUESTED** [review]
Thank you @adimitui for your work on this PR. I've left suggestions inline plus the following:
- **GLOBAL:** Please change all file names to lower case
- **MISSING DOCS:** Please add the docs for the 3 methods + 1 action supported by the tenantReference resource type. Then surface tenantReference and its methods in the TOC.
- Q: Does inboundSharedUserProfiles support the following actions - checkMemberObjects, checkMemberGroups, getMemberObjects, and getMemberGroups? If yes, we'll need to include the required references in the resepctive docs. 
- Please document the two new permissions, CrossTenantUserProfileSharing.Read, CrossTenantUserProfileSharing.Read.All, to the permissions reference table (I think under cross tenant access permissions is OK) and add a changelog object for each.
- I'm having a bit of trouble understanding who calls the exportPersonalData action. Left a Q for you on the action's page.
- Let's group all these APIs into a single sub-node in the TOC. You can call it "Cross-tenant shared user profiles". I think this will help with discoverability for all XTA-related APIs. Please feel free to suggest otherwise or share if this is a misrepresentation.


I'll ping you internally to work on an overview article for this API.

**Comment by FaithOmbongi** [comment]
The API docs work has been taken over via #20553 . Removing the changes requested label and closing this PR to avoid unactionable notifications.

**Pull request closed without merging**